### PR TITLE
Implements #356 (a) Added support for negative pageValidators

### PR DESCRIPTION
**Pull request by ibugaienko**
- [X] By placing an `X` in the preceding checkbox, I verify that I am a PayPal employee or
I have signed the [Contributor License Agreement](https://github.com/paypal/SeLion#contributing)

---
This pull request addresses problem discussed in #356 (1st solution)

### CODE REVIEW

File: client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      containerButton:
+        locators:
+          US: ".//input"
+pageTitle:
+  US: "TestPage Generated By JavaScript"
+pageValidators: [
+  hiddenButton.isNotVisible,
+  "!hiddenButton.isVisible",
+  "!continueButton",
+  hiddenButton.is!Visible,
```
Comment (mach6) [inline client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml]: Don't think we need to support the `!` in the middle of these two (`is` and `Visible`) words. :) 

It just looks odd.

File: client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  hiddenButton.!isVisible,
+  hiddenButton.isVisibleNot,
```
Comment (mach6) [inline client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml]: Same feedback here.. `isVisiibleNot` just looks odd.. I don't think we need to support this.

File: client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  hiddenButton.isVisible!,
```
Comment (mach6) [inline client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml]: Same feedback here... The `!` on the end of `isVisible` looks odd. I don't think we need to support this

**Comment by mach6** [comment]
@ibugaienko all in all I like this.. Just had the couple small points of feedback

File: client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml (hunk 2 above)
Comment (ibugaienko) [reply client/src/test/resources/GUIData/paypal/TestPageNegativeValidator.yml]: hiddenButton is visible. NOT!
Looks a bit like a not-joke.

**Comment by ibugaienko** [comment]
@mach6 , thank you, great feedback!
As the code written here, any of these will work, which is why I decided to include them all (even most odd ones) in the test. I will update the code checking `action` to be more picky.

**Comment by mach6** [comment]
@ibugaienko  any update on this?

**Comment by ibugaienko** [comment]
@mach6 , sorry to keep you waiting, I have updated the code, currently only below syntax is supported:
`hiddenButton.isNotVisible`
`"!hiddenButton.isVisible"`
`"!continueButton"`

Please let me know what you think.

**Pull request merged**